tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "allowJs": true,
+        "allowJs": false,
```

Comment:
Might as well remove this as the default is false and we don't use this anymore (was a leftover from me migrating to TS)